src/fetch_package.js

```diff
@@ -140,6 +140,22 @@ export default async function fetchPackage(
         return fetcher
             .use(superagentCache)
             .expiration(24 * 60 * 60)
+            .pruneKey((key) => ({
```

Comment:
This may seem a little extra, but could you split this out into its own function with tests? I think it would be helpful to document the edge cases of what does and does not expect to be modified, just to be sure we're covering what we think we are.